Bottom: 680fc1a29c1d6b271ded6c85bb6f2d6c5454d5a6
Top:    ace1b28254a186417363321fa229a3a2b965e355
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-05-08 11:36:58 +0100

Refresh of work-in-progress-on-subcomposi

---

diff --git a/src/hid/gtk/gtkhid-gl.c b/src/hid/gtk/gtkhid-gl.c
index 1a75bd3..075e940 100644
--- a/src/hid/gtk/gtkhid-gl.c
+++ b/src/hid/gtk/gtkhid-gl.c
@@ -135,6 +135,9 @@ ghid_end_layer (void)
 {
   render_priv *priv = gport->render_priv;
 
+  /* Flush out any existing geoemtry to be rendered */
+  hidgl_flush_triangles (&buffer);
+
   /* Relinquish any bitplane we previously used */
   hidgl_return_stencil_bit (priv->subcomposite_stencil_bit);
   priv->subcomposite_stencil_bit = 0;
